charts/newrelic-infrastructure-beta/templates/ksm/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.ksm.enabled -}}
```

Comment:
While these make sense conceptually, I'm not sure if we should let users disable `ksm` and/or `kubelet` components. If they do, cluster explorer (and any other experience we build in the future) will certainly not work, and the UI will return unmeaningful errors.

If we chose to allow users to flip this switch we should at least print a warning in `NOTES.txt`. WDYT?